### DISCUSSION TITLE
Here's how to set up autocomplete inside fenced code blocks in neovim

### DISCUSSION THREAD

**Discussion by llimllib** [observablehq/framework#892] [Show and tell]
I [wrote a short bit](https://notes.billmill.org/computer_usage/vim/otter.nvim.html) about how to use [otter.nvim](https://github.com/jmbuhr/otter.nvim/) to implement code completion inside the fenced js blocks of markdown documents.

It includes links to my config which you're free to copy.

Just wanted to share here in case other people were looking for this!

(I assume the same is possible in vs code? Share here if you have instructions for your editor!)

![image](https://github.com/observablehq/framework/assets/7150/47dcf85f-e338-4eba-81e1-09657452f673)

**Comment by wetlandscapes** [comment]
Sweet! Would be super cool to have this in vscode, too.
